war/WEB-INF/pages/errorPage.jsp

```diff
@@ -0,0 +1,23 @@
+<%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@taglib uri="http://www.springframework.org/tags/form" prefix="form"%>
+<% response.addHeader("Refresh","5"); %>
+<c:url var="resources" value="/resources/theme1" scope="request" />
+
+<html>
+    <head>
+        <link href="${resources}/css/core.css" rel="stylesheet">
+    </head>
+    <body>
+        <abb_h1>The Error Page</abb_h1><br>
+        <abb_h2>${input_message}</abb_h2>
+    </body>
+  
+        <form:form method="GET" action="/ABBJavaDev-SpringMVC/back">
```

Comment:
you can use /ABBJavaDev-SpringMVC/ instead of /ABBJavaDev-SpringMVC/back so you won't see the /back url just like the original index page